test/unit/update_versions_file_test.rb

```diff
@@ -8,18 +8,25 @@ class UpdateVersionsFileTest < ActiveSupport::TestCase
     Gemcutter::Application.load_tasks
   end
 
+  def update_versions_file
+    freeze_time do
+      @frozen_time = Time.now # rubocop:disable Rails/TimeZone
```

Comment:
I am guessing rubocop doesn't warn about `Time.now.iso8601`. Why not use that for @frozen_time?

Reply:
That's clever! updated